extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -257,6 +262,42 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * If requestsFrameUrl (and optionally dataHash as well) are specified in
+   * config, check whether third-party iframe already exists, and if not,
+   * create it.
+   * @private
+   */
+  handleRequestsFrameUrl_() {
+    if (!this.config_['requestsFrameUrl']) {
```

Comment:
Yes. I moved it to transport in the plane yesterday, and am currently testing before checking in that change.